hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaDistributionSetManagement.java

```diff
@@ -337,11 +338,18 @@ public DistributionSet unassignSoftwareModule(final long setId, final long modul
     }
 
     @Override
-    public Page<DistributionSet> findByDistributionSetFilter(final Pageable pageable,
+    public Slice<DistributionSet> findByDistributionSetFilter(final Pageable pageable,
             final DistributionSetFilter distributionSetFilter) {
         final List<Specification<JpaDistributionSet>> specList = buildDistributionSetSpecifications(
                 distributionSetFilter);
-        return convertDsPage(findByCriteriaAPI(pageable, specList), pageable);
+        return convertDsPage(findByCriteriaAPIWithoutCount(pageable, specList), pageable);
```

Comment:
Is there a reason for not using the new helper method JpaManagementHelper:convertPage?